core/domain/entities/editor/blocks/EventAttendees.php

```diff
@@ -55,16 +60,19 @@ public function initialize()
         $this->setBlockType(self::BLOCK_TYPE);
         $this->setSupportedRoutes(
             array(
-                'EventEspresso\core\domain\entities\route_match\specifications\admin\EspressoEventEditor',
-                'EventEspresso\core\domain\entities\route_match\specifications\admin\WordPressPostsEditor',
+                'EventEspresso\core\domain\entities\route_match\specifications\admin\EspressoPostTypeEditor',
+                'EventEspresso\core\domain\entities\route_match\specifications\admin\WordPressPostTypeEditor',
                 'EventEspresso\core\domain\entities\route_match\specifications\frontend\EspressoBlockRenderer',
```

Comment:
I think the change to use `WordPressPostTypeEditor` here is good,  but in my review and comments in Slack I didn't request extending this to EE post types.  I can understand having this block exposed in the editors for Venue and Event, but I don't think it would be good to expose it to Attendees.  What was your reasoning behind this additional change?

Reply:
`EspressoPostTypeEditor` is probably a poor name because it won't actually match for the EE Attendees screen because it is using a non-standard URL schema. Here's the logic in the `EspressoPostTypeEditor` route match specification:

```php
        return $this->request->getMatch('espresso_*') !== false
            && (
                $this->request->getRequestParam('action') === 'edit'
                || $this->request->getRequestParam('action') === 'create_new'
            );
```

which will return true for any URL having a parameter name matching the pattern `espresso_*` where `action` equals `edit` or `create_new`.
and here's what a URL for the edit attendee screen looks like:

```
page=espresso_registrations&action=edit_attendee&post=988&edit_attendee_nonce=cb8526453c&return=contact_list
```

which won't match the above logic.

So I either have to:

- rename `EspressoPostTypeEditor` to something else (`EspressoEventAndVenueEditor` ?)
- change the logic in `EspressoPostTypeEditor` to also detect the editing of `EE_Attendee`s and then create a new route match spec for events and venues only.

think I might go for the latter

Reply:
bah ya forgot about needing to change this. And it's not the first time I've done this exact same thing too :(

looks like we are just using the translation methods everywhere else for this, so I'll just do that

Reply:
Ya I like the latter option too.